x-pack/plugins/apm/public/components/app/settings/agent_configurations/agent_configuration_create_edit/service_page/form_row_suggestions_select.tsx

```diff
@@ -41,7 +42,7 @@ export function FormRowSuggestionsSelect({
       <EuiFormRow label={fieldLabel}>
         <SuggestionsSelect
           customOptions={allowAll ? [ENVIRONMENT_ALL] : undefined}
```

Comment:
This still uses `ENVIRONMENT_ALL` as a custom option but the suggestion select is used for both services and environments.

Reply:
I created the generic `ALL_OPTION` https://github.com/elastic/kibana/pull/133616/commits/cf04974141e1241ec1c15a7ad627005007b1b6f6#diff-b23eb2353f30e68304959fb4d0def2c1f80c45d552c4d6f7fb4d8f5668ff185aL32